frontend/src/pages/account/PasswordReset.vue

```diff
@@ -45,6 +47,17 @@ export default {
         }
     },
     computed: mapState('account', ['settings', 'pending']),
+    watch: {
+        'input.password': function (v) {
+            if (this.errors.password && v.length >= 8 && zxcvbn(v).score >= 2) {
+                this.errors.password = ''
+            }
+        }
+    },
+    async mounted () {
+        const { default: zxcvbnImp } = await import('zxcvbn')
```

Comment:
why `import` in here? Why not import at the top level with others?

Reply:
Because it has to be loaded async (it's huge so need to only be loaded on demand) so I assumed it needs to be in an async function